lib/ccd/templates/problem_concern_act_condition_template.rb

```diff
@@ -10,10 +10,9 @@ def self.included(base)
       constraint 'mood_code', {:cardinality=>"1..1", :value=>"EVN"}
       
       # SHALL contain exactly one [1..1] templateId (CONF:16772) such that it
```

Comment:
Do we want to update this comment and/or the one below it? Same thought for other places we've made this change.

Reply:
What do you think we should change it to?

Reply:
It feels incomplete because it ends with 'such that it' but.. I looked it up a little and I guess it's okay to leave it as is 👍